src/MyHealth/Calendar.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { FunctionComponent, useEffect } from "react"
+import { Text, TouchableOpacity, View, StyleSheet } from "react-native"
+import dayjs from "dayjs"
+
+import { Spacing, Typography } from "../styles"
+import { GlobalText } from "../components"
+import { DayLogData } from "./symptoms"
+import DayIndicator from "./DayIndicator"
+import { DateTimeUtils } from "../utils"
+
+interface CalendarProps {
+  logDataHistory: DayLogData[]
+  onSelectDate: (logData: DayLogData) => void
+  selectedDay: DayLogData | null
+}
+
+const Calendar: FunctionComponent<CalendarProps> = ({
+  logDataHistory,
+  onSelectDate,
+  selectedDay,
+}: CalendarProps) => {
+  const lastMonth = dayjs().subtract(1, "month")
+  const title = `${lastMonth.format("MMMM")} | ${dayjs().format(
+    "MMMM",
+  )}`.toUpperCase()
+
+  useEffect(() => {
+    if (!selectedDay) {
```

Comment:
Do we need to autoselect the day for the user? if so, would it make sense to remove the null option for the selected day on the prop?